backend/app/literature/crud/reference_crud.py

```diff
@@ -205,6 +217,15 @@ def show(db: Session, curie: str):
             del editor['resource_id']
             del editor['reference_id']
 
+    if reference.merged_into_id:
+       reference_data['merged_into_reference_curie'] = db.query(ReferenceModel.curie).filter(ReferenceModel.reference_id == reference_data['merged_into_id']).first()[0]
+
+    if reference.mergee_references:
+        mergees = []
+        for mergee in reference.mergee_references:
+            mergees.append(mergee.curie)
+        reference_data['mergee_reference_curies'] = mergees
```

Comment:
var names are a bit confusing here. Can you rewrite this as `reference_data['mergee_reference_curies'] = [mergee.curie for mergee in reference.mergee_references]` and get rid of lines 224-226?